examples/all-models-metadata/src/index.tsx

```diff
@@ -24,13 +24,13 @@ interface OpenRouterModel {
     max_completion_tokens: number | null;
     is_moderated: boolean;
   };
-  per_request_limits: any | null;
+  per_request_limits: unknown;
 }
 
 // This interface is more generic to accommodate both OpenAI and OpenRouter responses
 interface ListModelsOutput {
   models: {
-    data: any[]; // Using any[] to accommodate different model structures
+    data: unknown[]; // Using any[] to accommodate different model structures
```

Comment:
nit: the comment is now outdated
```suggestion
    data: unknown[]; 
```